CanUr/Player.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEngine;
+using Assets.Pixelation.Example.Scripts;
+using System.Collections;
+
+public class Player : MonoBehaviour {
+	//REVOLUTIE VOOR DE HAAKJES PLZ!!!
+
+	public bool lockMouseMovement;
+	public MouseLook look;
+
+	public Transform weapon;
+
+    	public int health {
+		get;
+		set;
+	}
+
+	private Vector3 lastPos;
+    	private int hurtDelay = 0;
+
+	void Start() {
+		health = 5;
+		lastPos = transform.position;
+	}
+
+	void Update() {
+        if (hurtDelay > 0) hurtDelay++;
+        if (hurtDelay > 20) hurtDelay = 0;
+
+        if (lockMouseMovement) look.enabled = false;
+		else {
+			look.enabled = true;
+			if(weapon.GetComponent<Weapon>()) weapon.GetComponent<Weapon>().WeaponUpdate();
+		}
+	}
+
+	void FixedUpdate() {
+		if(!lockMouseMovement && weapon.GetComponent<Weapon>()) weapon.GetComponent<Weapon>().FixedWeaponUpdate();
+	}
+
+	public bool isMoving() {
```

Comment:
Ik twijfel of dit Pascal-case moet of niet. Ik keur het goed in dit geval.

Reply:
Yes dit moet in PascalCase!